src/app/metadata/services/datasetSummary.service.ts

```diff
@@ -0,0 +1,67 @@
+/**
+ * Short module description
+ * @copyright The GHGA Authors
+ * @license Apache-2.0
+ */
+
+import { HttpClient } from '@angular/common/http';
+import { computed, inject, Injectable, resource, signal, Signal } from '@angular/core';
+import { ConfigService } from '@app/shared/services/config.service';
+import { firstValueFrom } from 'rxjs';
+import { DatasetSummary, emptyDatasetSummary } from '../models/dataset-summary';
+
+/**
+ * Dataset summary service
+ *
+ * This service provides the functionality to fetch dataset summary data from the server.
+ */
+@Injectable({
+  providedIn: 'root',
+})
+export class DatasetSummaryService {
```

Comment:
I would not create a separate service for each endpoint, but bundle similar things together. In this case, I would just create one `MetadataService` that would contain all the metadata related stuff that also use the same backend service. It should also provide the dataset detail later.

You can argue that the `MetadataSearchService` should stay separate, since it's also using a different backend service, but at least the other services should be merged together.

Reply:
Done 👍